images/dotcms/ROOT/srv/entrypoint.sh

```diff
@@ -23,15 +23,25 @@ if [[ "${1}" == "dotcms" || -z "${1}" ]]; then
 
     cd /srv/home
 
+    if [ ! -z "${WAIT_DB_FOR}" ]; then
```

Comment:
This too. Adding some extra time to wait for the database in fully usable.
In Oracle it takes a while.